docs/fsharp/using-fsharp-on-azure/table-storage.md

```diff
@@ -41,9 +54,13 @@ Add the following `open` statements to the top of the `tables.fsx` file:
 
 [!code-fsharp[TableStorage](../../../samples/snippets/fsharp/azure/table-storage.fsx#L1-L5)]
 
-### Get your connection string
+### Get your Azure Storage connection string
+
+If you are connecting to Azure Storage Table service, you'll need your connection string for this tutorial. You can copy your connection string from the Azure portal. For more information about connection strings, see [Configure Storage Connection Strings](/azure/storage/storage-configure-connection-string).
```

Comment:
nit: [use contractions](https://docs.microsoft.com/en-us/style-guide/word-choice/use-contractions)

If you are -> If you're